tox.ini

```diff
@@ -8,7 +8,11 @@ deps =
     -rrequirements.txt
     pytest-cov
 passenv = TRAVIS* PG* SQLALCHEMY_DATABASE_TEST_URI PERSISTENCE_DIR FLASK_APP SECRET_KEY
-commands = py.test --cov portal --cov-report xml:"{toxinidir}/coverage.xml" []
+whitelist_externals = /bin/bash
```

Comment:
Can you undo the changes to this file (so it's the same as the version in `develop`)?

While you were gone I merged [this PR](https://github.com/uwcirg/true_nth_usa_portal/pull/2319) which deletes the list of files in `tox.ini` and moves them to each individual test file (so I can run the same command for `py27` and `py34` tox environments)

Reply:
Gotcha, will do